src/utils/viewer/viewer.cpp

```diff
@@ -769,6 +798,44 @@ void Viewer::drawEvent() {
   redraw();
 }
 
+void Viewer::moveAndLook(int repetitions) {
+  if (repetitions <= 2) {repetitions = 1;}    // for visual smoothness, for small deviations around 60fps, we default to repetitions=1.
```

Comment:
We discussed offline and agreed this isn't necessary, so I expect you'll remove this line.